content/references/block-utils.md

```diff
@@ -244,3 +245,19 @@ utils.seo.addExtraTag('twitter:title', 'Your Page Title')
 | getDataProps utils | Block props |
 | ------------------ | ----------- |
 | &check;            | &cross;     |
+
+### 'storeUrl'
+
+This is a string value equal to the base URL of the current store.
+
+#### 'seo' Usage
+
+```javascript
+utils.storeUrl === "https://www.my-store.com"
```

Comment:
Maybe update this for constructing an API url, which is the only real use.

Reply:
@cameron-jones-volusion thoughts?

Reply:
Could it be used to construct links to elsewhere on the store? Like `${utils.storeUrl}/cart`? Otherwise, I can update the example; can you give an example of what an API URL looks like?

Reply:
It could be, but there's not a lot of reason to do that when `/cart` works just fine and will function on preview URLs without navigating you to the live storefront. 

API url example: `${utils.storeUrl}/api/v2/storefront/products/${productCode}`

Reply:
@dallonf maybe comment out `utils.storeUrl === "https://www.my-store.com"` to show that it's part of the documentation instead of executable JS? And update the API url example to be fully executable: 
```
const exampleApiUrl = `${utils.storeUrl}/api/v2/storefront/products/${productCode}`;
```